prime-router/src/test/kotlin/common/ReportNodeBuilder.kt

```diff
@@ -189,6 +208,7 @@ class ReportNodeBuilder {
         }
     }
     lateinit var theAction: TaskAction
```

Comment:
why are some of these lateinit and some of these are nullable vars?

Reply:
Good point! I'm going to take a 2nd look; on first (2nd) glance they should probably both be nullable since null is a valid state in either case.